.travis.yml

```diff
@@ -16,26 +13,32 @@ addons:
 
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - VERSION="saas-18" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
   - TRANSIFEX_USER='transbot@odoo-community.org'
   - secure: Z06mZCN+Hm3myqHSOZpOOk1pd4oq1epAWZv6m9OX2bTNHbhyOVOGK6JWWsnDm/3DUCN1ZeLtSGOl9bvQfMa8ahQHA80MkLL16YlTvQV59Lh+L2gAYmxX+ogJCJgeQSVAXlGLscgkADCu/HzDlmatrDeROMtULn5i23j2qcyUNyM=
   - COUNTRY="MX"
 
-  matrix:
-  - LINT_CHECK="1"
-  - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-
-virtualenv:
-  system_site_packages: true
+matrix:
+  include:
+    - python: 2.7
+      env:
+        LINT_CHECK="1"
+    - python: 3.4
+      env:
+        TRANSIFEX="1"
+    - python: 3.4
+      env:
+        TESTS="1" ODOO_REPO="odoo/odoo"
+    - python: 3.4
+      env:
+        TESTS="1" ODOO_REPO="OCA/OCB"
 
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --single-branch --depth=1 https://github.com/vauxoo-dev/maintainer-quality-tools.git -b master-vx-support-python3-jesuszapata ${HOME}/maintainer-quality-tools
```

Comment:
If we are using 2.7 for pylint why we need to use this branch?

Reply:
@moylop260 
The previous ran over travis not show the mistake https://travis-ci.org/Vauxoo/server-tools/jobs/280228437#L4606

But now the build on travis show the follow mistake 

```
From https://github.com/Vauxoo/server-tools
 * branch            saas-18    -> FETCH_HEAD
fatal: ambiguous argument 'origin/saas-18': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

https://travis-ci.org/Vauxoo/server-tools/jobs/280429804#L4565